tests/test_collector.py

```diff
@@ -30,15 +39,17 @@ def path_artifact():
     return artifact
 
 
-def test_collector(command_echo, passwords_file, outputs, fake_partitions):
+def test_collector(command_echo, passwords_file, passwords_file_info, outputs, fake_partitions):
     collector = Collector(get_operating_system())
 
     collector.register_source(command_echo, command_echo.sources[0])
     collector.register_source(passwords_file, passwords_file.sources[0])
+    collector.register_source(passwords_file_info, passwords_file_info.sources[0])
     collector.collect(outputs)
 
     assert outputs.add_collected_file.call_count == 1
     assert outputs.add_collected_command.call_count == 1
+    assert outputs.add_collected_file_info.call_count == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/fastir-build/fastir_artifacts/pullRequest?prid=5744135)